src/System.Runtime.Numerics/tests/ComplexTests.cs

```diff
@@ -226,11 +226,32 @@ public static IEnumerable<object[]> Abs_Advanced_TestData()
                 foreach (double invalidImaginary in s_invalidDoubleValues)
                 {
                     yield return new object[] { RandomPositiveDouble(), invalidImaginary, Math.Abs(invalidImaginary) }; // Invalid imaginary
-                    yield return new object[] { invalidReal, invalidImaginary, (double.IsInfinity(invalidReal) || double.IsInfinity(invalidImaginary)) ? double.PositiveInfinity : double.NaN }; // Invalid real, invalid imaginary
+                    yield return new object[] { invalidReal, invalidImaginary, (double.IsNaN(invalidReal) || double.IsNaN(invalidImaginary)) ? double.NaN : double.PositiveInfinity }; // Invalid real, invalid imaginary
```

Comment:
@mellinoe: Done in PR update, but please let me know if I did it right. I don't see any way to skip MemberData for one Theory method, only to skip the entire Theory method. So I effectively created two Theory methods for both Abs and Sqrt, each with their own MemberData, and skipped one for each.

I must admit I don't fully understand how netfx and corefx interact. It appears that all these tests are being run against netfx, even though the corefx implementations aren't being moved into netfx?! That seems like a recipe for constant breakage, and aren't these supposed to become the netfx implementations anyway?